samples/net/sockets/echo_server/Kconfig

```diff
@@ -58,4 +58,12 @@ config NET_SAMPLE_PSK_HEADER_FILE
 	  Name of a header file containing a
 	  pre-shared key.
 
+config NET_SAMPLE_CERTS_WITH_SC
+	bool "Signed certificates"
+	default n
+	depends on NET_SOCKETS_SOCKOPT_TLS
+	help
+	 Enable this flag, if you are interested to run this
```

Comment:
Small style nit here, the help text should be intended by two chars, now there is one

Reply:
Done.